protocol/contracts/amo/Ramos.sol

```diff
@@ -390,6 +391,9 @@ contract RAMOS is Ownable, Pausable {
                 unchecked {
                     receivedAmount = stable.balanceOf(address(this)) - stableAmountBefore;
                 }
+                if (receivedAmount > 0) {
+                    stable.safeTransfer(msg.sender, receivedAmount);
```

Comment:
Perhaps have the receiver as a function parameter? I've found it's useful to save gas in integrations later (rather than 2 transfers)

Should we add this to withdrawStable too for symmetry?

Reply:
Good. Will add this to `withdrawStable` too.